benchmarks/utils.py

```diff
@@ -0,0 +1,336 @@
+# type: ignore
```

Comment:
Add docstring "We are generating an artificial dataset that is big, and we need to do this in a faster way than before."